From c93188b6c6973932d2adaa52e6a4920db13b4e62 Mon Sep 17 00:00:00 2001
Message-Id: <c93188b6c6973932d2adaa52e6a4920db13b4e62.1350493760.git.minovotn@redhat.com>
From: Jeffrey Cody <jcody@redhat.com>
Date: Wed, 17 Oct 2012 05:59:14 +0200
Subject: [PATCH 01/35] raw-posix: don't assign bs->read_only

RH-Author: Jeffrey Cody <jcody@redhat.com>
Message-id: <98aac1ecccf74980a1b76d9d2a66bdd9a896c513.1350447475.git.jcody@redhat.com>
Patchwork-id: 43259
O-Subject: [RHEL6.4 qemu-kvm PATCH v4 01/35] raw-posix: don't assign bs->read_only
Bugzilla: 767233
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Eric Blake <eblake@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>

From: Christoph Hellwig <hch@lst.de>

bdrv_open already takes care of this for us.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Acked-by: Kevin Wolf <kwolf@redhat.com>
Signed-off-by: Aurelien Jarno <aurelien@aurel32.net>
(cherry picked from commit 6e0a47aae03c6646c3094b2b9a7625d8dcb36e55)
Signed-off-by: Jeff Cody <jcody@redhat.com>
---
 block/raw-posix.c | 1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 block/raw-posix.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/block/raw-posix.c b/block/raw-posix.c
index 477ced2..662aef5 100644
--- a/block/raw-posix.c
+++ b/block/raw-posix.c
@@ -184,7 +184,6 @@ static int raw_open_common(BlockDriverState *bs, const char *filename,
         s->open_flags |= O_RDWR;
     } else {
         s->open_flags |= O_RDONLY;
-        bs->read_only = 1;
     }
 
     /* Use O_DSYNC for write-through caching, no flags for write-back caching,
-- 
1.7.11.7

